selfswab/tasks.py

```diff
@@ -82,6 +83,10 @@ def poll_meditech_api_for_results():
                     if result.get("verifyDateTime"):
                         registration.authorized_timestamp = result.get("verifyDateTime")
 
+                    if not registration.pdf_media_id and result.get("pdf_path"):
+                        content = requests.get(result["pdf_path"]).content
+                        registration.pdf_media_id = upload_turn_media(content)
+
```

Comment:
TODO:
On this flow start we need to send a flag to rapidpro. In the flow we need to notify admin if there is a result but no pdf